kolla-ansible/ansible/roles/triliovault/tasks/config.yml

```diff
@@ -25,6 +25,20 @@
   when:
     - inventory_hostname in groups[triliovault_datamover_group]
 
+- name: copy fuse conf file
+  become: true
+  template:
+    src: fuse.conf
+    dest: "/etc/fuse.conf"
```

Comment:
Permission and ownership should be like this.  42436 is nova user and group id.
      file { "/etc/triliovault-wlm/fuse.conf":
          ensure  => present,
          content => template('trilio/fuse.conf.erb'),
          owner  => '42436',
          group  => '42436',
          mode   => '0644',

Reply:
it's resolved.